docs/range-proof-protocol.md

```diff
@@ -1,190 +1,315 @@
-The `range_proof` module contains API for producing a range proof for a given integer value.
+The `aggregated_range_proof` module contains an API for producing a proof that multiple integer values are within a certain range.
```

Comment:
This should probably refer to `range_proof` now that the `aggregated_range_proof` module is gone.

Reply:
👍